static/js/composebox_typeahead.js

```diff
@@ -181,6 +181,11 @@ function autocomplete_checks(q, char) {
         return false;  // char doesn't appear, or too far back
     }
 
+    // Don't autocomplete if there is a space following a '#'
+    if (char === "#" && q.charAt(last_at + 1) === " ") {
```

Comment:
We should add a test where `#` is at the end of string, just to make sure the `last_at + 1` lookup doesn't crash anything.